index.js

```diff
@@ -327,6 +327,10 @@ exports.importPublic = function (publicKey) {
 exports.ecsign = function (msgHash, privateKey) {
   var sig = secp256k1.sign(msgHash, privateKey)
 
+  if (sig.recovery === undefined) {
```

Comment:
In what case can this be `undefined`?

I'd count that as a failure and as such I would throw an error.

Reply:
I followed what was happening and it's not the library's fault. 
My webpack was making sekp256k1 to load elliptic@3.0.3 instead of the elliptic@6.2.3 specified in the dependency.
So this is not a bug of the library. Please close the PR.